public/scripts/worktrollyUI.js

```diff
@@ -85,7 +86,8 @@ $("#editTask").ready(function() {
 });
 
 $("#logWork").ready(function() {
-    $("#logWork").hide(0);
+    // $("#logWork").hide(0);
+    fillDataIntoLogWorkPage("D1", "qwe", 1, 12, 0, 0);
```

Comment:
Please remove this line, we don't need this anymore, we just did it for testing purpose.

Reply:
ok